src/ol-layers-logic/ClusteredFeaturesLayer.ts

```diff
@@ -134,12 +135,20 @@ abstract class ClusteredFeaturesLayer {
                         const splitted = p.split('_')
                         p = splitted[0].charAt(0).toUpperCase() + splitted[0].slice(1) + ' ' + splitted[1].toUpperCase()
                     }
+                    const osm_id = value
                     const types_mapping = {
                         n: 'node',
                         w: 'way',
                         r: 'relation'
                     }
-                    value = '<a target="_blank" href="https://www.openstreetmap.org/' + types_mapping[data_type] + '/' + value + '">' + value + '</a>';
+                    const osm_url = 'https://www.openstreetmap.org/' + types_mapping[data_type] + '/' + osm_id;
+                    // https://wiki.openstreetmap.org/wiki/JOSM/RemoteControl
+                    // Link target is a hidden iframe, otherwise browsers open a new tab
+                    const bbox = map.getView().calculateExtent(map.getSize());
+                    const josm_url = `http://localhost:8111/load_and_zoom?left=${bbox[0]}&right=${bbox[2]}&top=${bbox[3]}&bottom=${bbox[1]}&select=${types_mapping[data_type]}${osm_id}&zoom_mode=download`;
+                    const josm_help = 'JOSM must be running and JOSM remote control plugin must be enabled for this to work';
+                    value = `<a target="_blank" href="${osm_url}">${osm_id}</a>`
+                            + ` <a href="${josm_url}" target="hiddenIframe" title="${josm_help}">(edit in JOSM)</a>`;
```

Comment:
Can we have an icon instead of the text 'edit in JSOM', maybe something like [OSM Inspector](https://tools.geofabrik.de/osmi/) has? Because the next feature request will be to also add an edit link to Id and the space will become very sparse. ;)